[ADD] accounting: new doc about EU intra-community distance selling (#1047)
odoo/documentation: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst

===================================
EU intra-community distance selling
===================================

**Distance selling within the European Union** implies that organizations that sell goods and
services cross-border to a private consumer (B2C) in an EU Member State must ensure that the VAT is

> paid to the Member State in which the goods or services are delivered.

Author: Is this better?
It is also more detailed.

```
**Distance sales within the European Union** include cross-border sales of goods and services to a
private consumer (B2C) in another EU Member State when the seller doesn't meet face-to-face with the
customer. Organizations must ensure that the VAT on distance sales is paid to the Member State in
which the goods or services are delivered.
```

> paid to the Member State in which the goods or services are delivered.

Reviewer: Much better!

.. note::
   This is also valid if your organization is located outside of the European Union.

While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to

> private EU consumers, it is also valid for regular cross-border sales.

Author: ```
While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
private EU consumers, it is also valid for mail order sales and telesales.
```

> private EU consumers, it is also valid for regular cross-border sales.

Reviewer: Looks good to me!

> The Union **One-Stop Shop (OSS)** is an online portal that businesses can use to register for the
> OSS and declare their transactions.

Author: ```suggestion
The Union **One-Stop Shop (OSS)** is an online portal where businesses can register for the OSS and
declare their intra-community distance sales.
```

The **EU intra-community distance selling** feature helps comply with this regulation by creating
and configuring new :doc:`fiscal positions <fiscal_positions>` and :doc:`taxes <taxes>` based on
your company's country.

.. seealso::
   - `European Commission: OSS | Taxation and Customs Union
     <https://ec.europa.eu/taxation_customs/business/vat/oss_en>`_

.. _eu_distance_selling/configuration:

Configuration
=============

Go to :menuselection:`Accounting/Invoicing --> Settings --> Taxes`, then enable **EU intra-community
distance selling** (or **EU Digital Goods VAT** if you created your database before July 1, 2021),
and *Save*.

.. important::
   Please :ref:`upgrade the module <general/upgrade>` `l10n_eu_service` if you already installed it
   before **July 1, 2021**, or if you activated the feature **EU Digital Goods VAT** in the

>    Accounting settings.

Reviewer 2: Perhaps also a short mention that it means you need to update your Odoo instance it's source code if you are on premise too? I think quite some people wouldn't know that. You can link to the original docs about updates/deployments.

Author: Thanks for the feedback @Yenthe666 

I'll probably add this in the "general" doc, in the "upgrade" section referenced here rather than explain it in this documentation. This way, we don't overload all docs that require an upgrade, but if people need more information, they'll find it in the "upgrade apps and modules" section.

Since it's a change in 13.0, I'll do it with another PR.

Reviewer 2: Sounds good, thanks :)

>    Accounting settings.

Author: it simply means that the module must be upgraded at this date.

the only thing is that it has a different name in the accounting app if the database was created before July 1, 2021.

Fiscal Positions and Taxes
==========================

Once enabled, the feature creates all the necessary taxes and fiscal positions needed for each EU
member state, based on your company's country.

> We highly recommend you check that the proposed mapping is suitable for the products and services
> you sell before using it.

Reviewer: ```suggestion
We highly recommend checking that the proposed mapping is suitable for the products and services
you sell before using it.
```

===================================
EU intra-community distance selling
===================================

**Distance sales within the European Union** include cross-border sales of goods and services to a
private consumer (B2C) in another EU Member State when the seller doesn't meet face-to-face with the
customer. Organizations must ensure that the VAT on distance sales is paid to the Member State in
which the goods or services are delivered.

.. note::
   This remains true even if your organization is located outside of the European Union.

> While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to

Reviewer 3: ```suggestion
While this regulation mainly applies to :doc:`eCommerce <applications/websites/ecommerce>` sales to
```

Links can be clearer and stronger if they are relative from the root content directory :) (and will stay valid, even if you move the current file).

Author: Well, that's what I wrote initially but then @AntoineVDV  advised otherwise (in a different pull request)

-> https://github.com/odoo/documentation/pull/1044#discussion_r658545638

Reviewer 3: Well, he validated the opposite in https://github.com/odoo/documentation/pull/966 :thinking:.
If I remember correctly, we concluded sthg like "If there is more than three "../", then the link could be changed to an absolute one" ^^.
In this case, an absolute link is IMHO stronger than a relative one, but @AntoineVDV can have the final say :sweat_smile:

Reviewer 3: Forget it, we will use relative links to use a clear logic :ok_hand: (After discussion with ANV ^^)

Reviewer 4: To be clear, the logic would be: use an "absolute" path in the ref when you need to traverse the root anyway.

Although @Feyensv is right that our relative link may be less robust than an absolute one, this is no longer the case when the page you're editing is buried deep in the toctree and only has the first few nodes of its path in common with the page you're making a reference to. At this point, the slightest change in the ancestor nodes of the target page will break every single link.

In short, both approaches are valid but one works best in some cases and vice-versa. Nevertheless, we need a universal logic to handle cross-app references.
